src/app/views/censusdata/create.html

```diff
@@ -1,27 +1,18 @@
 <div class="container">
-    <div class="well row" >
-        <h4>Census participants please be as complete as possible:</h4>
-        <ul>
-            <li>Review form guidelines prior to census.</li>
-            <li>Always provide "number of adults and juveniles" data for species.</li>
-            <li>Always observe birds from at least 50 yards away.</li>
-        </ul>
-    </div>
-                
+    <div ng-include="'/app/views/censusdata/progressBar.html'"></div>
+    {{censusForm.step = 1}}
+
     <form class="form-horizontal" role="form"> 
         <div class="form-group panel panel-primary">
-            <div class="panel-heading">Site Information: Section One<span style="float:right;">Report violations to 1-800-792-4263</span></div>
-            <div class="panel-body">              
+            <div class="panel-heading">Site Information<span style="float:right;">Report violations to 1-800-792-4263</span></div>
```

Comment:
Let's remove that phone number. It will be needed elsewhere in Audubon's web site, but we don't need it in this demo site.